src/main/java/properties/Loader.java

```diff
@@ -0,0 +1,35 @@
+package properties;
+
+import java.io.*;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Loader {
+    private final Map<String, String> properties = new HashMap<>();
+
+    public Loader(String filePath) {
+        loadProperties(filePath);
+    }
+
+    private void loadProperties(String filePath) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                if (line.contains("=")) {

```

Comment:
it's possible to have comment starting with # and contains =
like

#here is the example key=value
color=red

Reply:
use:
if (!line.trim().isEmpty() && !line.startsWith("#"))